jquery.floatThead.js

```diff
@@ -705,6 +706,13 @@
             }
             left = tableOffset.left - windowLeft;
           }
+          if (top < 0) {
+            // in some fixed layout's the top offset can return a negative, in which case using
+            // the last known good value works 100%
+            top = goodTop;
```

Comment:
my might be better to simply ignore negative offsets. at the top of this fn as soon as you find a negative offset just return, it should have the same effect as doing this

i am still waiting for a jsfiddle that reproduces this before i merge it.